src/stores/video.ts

```diff
@@ -194,10 +259,11 @@ export const useVideoStore = defineStore('video', () => {
       fileName,
       vWidth,
       vHeight,
+      hash: recordingHash,
+      thumbnail: await extractThumbnail(),
     }
     unprocessedVideos.value = { ...unprocessedVideos.value, ...{ [recordingHash]: videoInfo } }
 
-    activeStreams.value[streamName]!.mediaRecorder!.start(1000)
```

Comment:
Why is this line being removed?

Edit: I think it is part of the next commit actually, right?

Reply:
The line has moved down inside the function

Reply:
Yes, but not on the same commit 😅

Reply:
You're right. This must be fixed